changelog/add-6051-tooltip-negative-pending-balance

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Shows link to documentation in the tooltip when pending balance is negative.
```

Comment:
I've suggested a minor change to the changelog entry to improve grammar:
```suggestion
Show a link to the documentation in the tooltip when the pending balance is negative.
```

Reply:
Thanks!